service/src/main/resources/api/openapi.yml

```diff
@@ -192,9 +192,10 @@ components:
           type: string
 
   securitySchemes:
-    bearerAuth:
-      type: http
-      scheme: bearer
+    oidc:
+      type: openIdConnect
```

Comment:
We might want the template to have the same fixes as Sam re the lack of support for `openIdConnect` in `openapi-generator`. 

https://github.com/broadinstitute/sam/pull/1430